### PR TITLE
Reestructuring rosie based on python packages

### PR DESCRIPTION
**Pull request by anaschwendler**
First steps towards making rosie more generic following
[this gist](https://gist.github.com/c98a91a9447b9404abd3b449daba50ec) and [this part](https://docs.python.org/3/tutorial/modules.html#packages) of python documentation.
Thanks to @jtemporal for the time the help to start it.

🎉

### CODE REVIEW

**Comment by cuducos** [comment]
CI tells us we have lost +20% of coverage. This _probably_ is because we haven't gathered all package code into a single module (i.e. `rosie`) — a top-level package as mentioned in the Python docs linked when opening the PR. When we ask for coverage of `rosie` module it's basically `rosie.py` file.

In the gist I suggested a `rosie` module as a top-level package. This is at the root of the repo and within it there would be a `core` module, a `ceap` module etc. So we have all package code within a single module (`rosie`, the top-level package). The way the PR is at this point makes the code sparse.

To be clear, what I suggested (updating the gist) is:

```
|- rosie
|   |- core
|   |   |- classifiers
|   |   |   |- …
|   |   |- tests
|   |   |   |- fixtures
|   |   |   |   |- …
|   |   |   |- …
|   |   |- __init__.py  # pipeline de execução
|   |- ceap
|   |   |- classifiers
|   |   |   |- …
|   |   |- tests
|   |   |   |- fixtures
|   |   |   |   |- …
|   |   |   |- …
|   |   |- __init__.py  # baixa dados
|   |   |- adapter.py
|   |   |- settings.py  
|   |- ceaps  # ceap senado
|       |- classifiers
|       |   |- …
|       |- tests
|       |   |- fixtures
|       |   |   |- …
|       |   |- …
|       |- __init__.py  # baixa dados
|       |- adapter.py
|       |- settings.py 
|- .codeclimate.yml
|- .coveragerc
|- …
```

But this is how it's suggested:

```
|- ceap
|   |- classifiers
|   |   |- …
|   |- tests
|   |   |- fixtures
|   |   |   |- …
|   |   |- …
|   |- __init__.py  # baixa dados
|   |- adapter.py
|   |- settings.py  
|- ceaps  # ceap senado
|   |- classifiers
|   |   |- …
|   |- tests
|   |   |- fixtures
|   |   |   |- …
|   |   |- …
|   |- __init__.py  # baixa dados
|   |- adapter.py
|   |- settings.py 
|- .codeclimate.yml
|- .coveragerc
|- …
```
IMHO we should go for a top-level package.

File: ceap/tests/fixtures/irregular_companies_classifier.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cnpj_cpf,situation,situation_date,issue_date
-02989654001197,ABERTA,2013-01-03,2013-01-30
+02989654001197,ABERTA,201 3-01-03,2013-01-30
```
Comment (luizcavalcanti) [inline ceap/tests/fixtures/irregular_companies_classifier.csv]: is this intentional?

Comment (anaschwendler) [inline ceap/tests/fixtures/irregular_companies_classifier.csv]: oh no! thanks for notice! will change it soon

File: rosie.py [hunk 2]
```diff
@@ -18,15 +18,15 @@ def help():
 
 
 def run():
-    import rosie
+    from rosie.ceap import classifiers
```
Comment (Irio) [inline rosie.py]: If we call it "CEAP", we would need the document and always explain what the abbreviation means. Up to this point, Rosie only works with Brazilian data, so would be safe to call it just "chamber_of_deputies".

Comment (anaschwendler) [inline rosie.py]: great, will be changing it soon

Comment (cuducos) [inline rosie.py]: Either way documentation is important. I believe each module should have at least a description at the beginning of module's `__init__.py` ([for example](http://dolphm.com/pep257-good-python-docstrings-by-example/)).

**Pull request merged**